packages/eslint-plugin/package.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "@redwoodjs/eslint-plugin",
+  "version": "5.0.0",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/redwoodjs/redwood.git",
+    "directory": "packages/eslint-plugin"
+  },
+  "license": "MIT",
+  "main": "dist/index.js",
+  "types": "./dist/index.d.ts",
+  "files": [
+    "dist"
+  ],
+  "scripts": {
+    "build": "yarn build:js && yarn build:types",
+    "build:js": "babel src -d dist --extensions \".js,.jsx,.ts,.tsx\"",
+    "build:types": "tsc --build --verbose",
+    "build:watch": "nodemon --watch src --ext \"js,jsx,ts,tsx\" --ignore dist --exec \"yarn build\"",
+    "prepublishOnly": "NODE_ENV=production yarn build",
+    "test": "jest src",
+    "test:watch": "yarn test --watch"
+  },
+  "devDependencies": {
+    "@babel/cli": "7.21.5",
+    "@types/eslint": "8",
+    "@types/estree": "1.0.1",
+    "jest": "29.5.0",
+    "prettier": "2.8.8",
+    "typescript": "5.1.3"
+  },
+  "gitHead": "3905ed045508b861b495f8d5630d76c7a157d8f1",
+  "dependencies": {
+    "@babel/core": "7.22.1",
+    "@babel/runtime-corejs3": "7.22.3",
+    "core-js": "3.30.2",
```

Comment:
@Tobbe I don't think you need to build this package with Babel and core-js; I can push a commit simplifying things if you want?

Reply:
Yes please 🙂

Reply:
Done in https://github.com/redwoodjs/redwood/pull/8612/commits/f51a72af50d4df4a43155c084e293fa01cb6739a